code/modules/admin/view_variables/modify_variables.dm

```diff
@@ -193,9 +193,10 @@ GLOBAL_PROTECT(VVpixelmovement)
 		else
 			variable = L[index]
 			//EXPERIMENTAL - Keep old associated value while modifying key, if any
-			var/found = L[variable]
-			if(!isnull(found))
-				old_assoc_value = found
+			if(IS_VALID_ASSOC_KEY(variable))
```

Comment:
this is true when variable is null

Reply:
And ?

Reply:
This was checking null before so I assumed null doesn't work as a key but I just tested and I suppose it does